spec/models/category_rspec.rb

```diff
@@ -3,4 +3,36 @@
 describe Category do
   it {should have_many(:videos)}
   it {should validate_uniqueness_of(:name)}
+
+  describe "#recent_videos" do
+    it "should return an empty array if no videos found" do

```

Comment:
Word `should` in example message is not bringing anything meaning to the sense of the whole spec. I recommend getting rid of this habit - Read more here - http://www.betterspecs.org/#should